app/views/cases/sar_internal_review/_case_details.html.slim

```diff
@@ -0,0 +1,27 @@
+.grid-row
+  div class="column-full #{case_details_for_link_type(link_type)}"
+    h2.request--heading
+      = translate_for_case(case_details, 'common', case_details_for_link_type(link_type))
+
+    table.report
+      tbody.sar-basic-details
+        tr.case-type
+          th Case type
+          td
+            = "#{case_details.pretty_type} "
+            = case_details.trigger_case_marker
+        tr.case-type
+          th Full name of subject
+          td
+            = "#{case_details.subject_full_name} "
+        tr.case-type
+          th Subject type
+          td
+            = "#{case_details.subject_type} "
+        tr.case-type
+          th Information requested on someone's behalf?
+          td
+            = "#{case_details.third_party_display} "
+
+         
+
```

Comment:
just for note, the field doesn't seems to show all those SAR IR fields,  the easiest way I would suggest is to make a copy of SAR case-detail page  rather than offender-SAR.


also it puzzled me a little bit, I am not sure you use this page as part of your SAR IR creation step or not as you use '_case_details.html.slim' , if do, then bear in mind, the information showed above will be linked SAR case, not SAR IR case,   and the name of the file needs to be changed, otherwise it gets really confusing, and I suspect, it will cause some trouble.

Reply:
Thanks that's a really good point - I might name it something like `_confirm_original_sar_for_sar_ir_case_step.rb` just to make it really obvious.

I'll update in this PR to make it clearer 👍